src/Core/CustomObjects/CustomProperty.cs

```diff
@@ -10,54 +10,67 @@ namespace Axe.Windows.Core.CustomObjects
 {
     public class CustomProperty
     {
+#pragma warning disable CA1720 // Identifier contains type name: name from JSON
+        private Guid _Guid;
```

Comment:
We follow the .NET pattern of starting property names with upper case, data members with lower case. As such, we should use `_guid`  (lower case 'g') instead of `_Guid` (upper case 'G')